.github/workflows/ci.yml

```diff
@@ -9,21 +9,8 @@ on:
 
 
 jobs:
-  bionic-ci:
+  focal-ci:
```

Comment:
Does it make sense to drop the ubuntu distro in the name and just call this job `ci`?

Reply:
I agree, done in cd88e88.